### ISSUE TITLE
Not working on Sublime 3

### ISSUE THREAD

**Issue by fynncfchen** [benedfit/SublimeLinter-contrib-jade-lint#4]
OS: Windows 7
node version: v4.2.6
pug-lint version: 2.1.3

pug-lint works on CLI, but not on Sublime 3.
I turn on the debug mode on sublimelinter and see nothing on status console when I open my `*.jade` files.

my `.pug-lintrc` setting

```
{
  "preset": "clock",
  "validateIndentation": 2,
  "validateAttributeQuoteMarks": "\"",
  "validateAttributeSeparator": ", "
}
```

**Comment by benedfit** [comment]
This issue was introduced in version `1.1.0`, and is fixed in version `1.1.1` by #3

**Comment by lixinyilalala** [comment]
It is doesn't be solved.... I really don't know what happened, and I followed what the https://github.com/benedfit/SublimeLinter-contrib-pug-lint said completely, but it doesn't work... it almost driven my crazy:( I installed the sublimeLinter, and then used the node to install the pug linter, then used the package control to install the contrib-pug-linter, I can find the path by "where pug-linter", and have enabled the pug-linter? Is there something I ignored? help! Thank you!
